docs/docsource/Operation-Architecture/Dashboards/Accessing_Cloud_Dashboards.rst

```diff
@@ -77,7 +79,7 @@ grep. For example, to find backbeat-api status, enter:
 
 ::
 
-    $ kubectl get pods | grep -i api
+    $ kubectl get pods | grep -i api
```

Comment:
While this is completely valid, it might be best to instruct people to use the integrated filter `kubectl get pods -l app=backbeat-api`.
The `-l` flag sorts by labels and is very versatile. Multiple filters can be applied at the same time via `-l key=value,key!=value..etc`. In practice it would look like `kubectl get pods -l release=my-zenko,app!=backbeat-api` which would grab all of zenko pods except for backbeat-api.

I'm not going to block the PR since this isn't the point of it but something we should look into changing.

Reply:
Happy to fix this when I'm dealing with the Helm typo. Any reason I shouldn't?

Reply:
Done.